wfe-core/src/main/java/ru/runa/wfe/chat/sender/SessionMessageSender.java

```diff
@@ -24,17 +24,27 @@
     private ObjectMapper chatObjectMapper;
 
     @Override
-    public void handleMessage(MessageBroadcast dto, Optional<Session> session) {
-        if (session.isPresent()) {
+    public void handleMessage(MessageBroadcast dto, Set<SessionInfo> sessions) {
+        if (sessions == null || sessions.isEmpty()) {
+            messageSender.handleMessage(dto, sessions);
+            return;
+        }
+
+        boolean isAnyBroadcastSending = false;
```

Comment:
```suggestion
        boolean isAnyBroadcastSent = false;
```
По-английски, думаю, так правильнее будет